hatServer/config.py

```diff
@@ -0,0 +1,4 @@
+WTF_CSRF_ENABLED = True
```

Comment:
I don't know if these two things are still necessary now that we're not using WTForms... Did you need to add them to change the configuration of hatServer?